buildbot/configure.py

```diff
@@ -252,6 +262,7 @@ def main():
     parser.add_argument("--enable-plugin", action='append', help="Enable SYCL plugin")
     parser.add_argument("--disable-fusion", action="store_true", help="Disable the kernel fusion JIT compiler")
     parser.add_argument("--add_security_flags", type=str, choices=['none', 'default', 'sanitize'], default=None, help="Enables security flags for compile & link. Two values are supported: 'default' and 'sanitize'. 'Sanitize' option is an extension of 'default' set.")
+    parser.add_argument("--default-openmp", action='store_true', help="Install default llvm openmp library")
```

Comment:
We already have an option for enabling LLVM's external projects: `--llvm-external-projects`.
You can enable OpenMP with `--llvm-external-projects=openmp`.

Reply:
Hi @bader,

Yes, you're right. I removed the option. I changed the `if-statement` as follows 
`if 'openmp' in llvm_external_projects:`

https://github.com/ravil-mobile/intel-llvm/blob/33fcd5db062c00ce01d8d40fd43eae959c8a7c61/buildbot/configure.py#L201-L205